docs/ref/templates/api.txt

```diff
@@ -201,10 +201,26 @@ Once you have a compiled :class:`Template` object, you can render a context
 with it. You can reuse the same template to render it several times with
 different contexts.
 
-.. class:: Context(dict_=None)
+.. class:: Context(dict_=None, autoescape=True, use_l10n=None, use_tz=None)
```

Comment:
`RequestContext` signature should also be updated in docs.

Reply:
Updated RequestContext